include/z64.h

```diff
@@ -49,6 +49,14 @@
 #define Z_PRIORITY_DMAMGR      16
 #define Z_PRIORITY_IRQMGR      17
 
+#define ALIGN8(val) (((val) + 7) & ~7)
```

Comment:
Should this align definition go in `include/macros.h` with the other aligns?

https://github.com/zeldaret/oot/blob/b41489c443943d273251b2497cac2ead2eea20ab/include/macros.h#L10-L14

Reply:
I'd rather not include macros.h in z64.h just for this

Reply:
Errr I meant the other way? Put `Align8` -> `macros.h`, not `AlighX` -> `z64.h`. Unless that's not what you meant

Reply:
If I put ALIGN8 in macros.h, I should really then include macros.h in z64.h to ensure it's visible everywhere that STACK is visible. But I'm reluctant to do this since macros.h is included after z64.h currently. Eventually much if not all of macros.h will be split out into separate headers, at which point this can be revisited.

Reply:
If there's a plan, then it's all good. For reference, MM has `ALIGN8` in `macros.h` (and has `ALIGN256` in a PR)